<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tâna vision</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Paramètres</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Commande</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,6,6">
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Appareil</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,2">
                  <item>
                   <widget class="QComboBox" name="cbCamera">
                    <item>
                     <property name="text">
                      <string>Aucune</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnSavePath">
                    <property name="text">
                     <string>Répertoire d'images</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Mode</string>
               </property>
               <widget class="QRadioButton" name="rbRGB">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>30</y>
                  <width>401</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>RGB (Normal)</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
               <widget class="QRadioButton" name="rbGRAY">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>60</y>
                  <width>401</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Gray (Niveau de gris)</string>
                </property>
               </widget>
               <widget class="QRadioButton" name="rbNegatif">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>90</y>
                  <width>401</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Négatif (couleur inversé)</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="box_action">
               <property name="minimumSize">
                <size>
                 <width>225</width>
                 <height>125</height>
                </size>
               </property>
               <property name="title">
                <string>Action</string>
               </property>
               <widget class="QRadioButton" name="rbRGM">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>53</y>
                  <width>341</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Reconnaissance de geste de la main</string>
                </property>
               </widget>
               <widget class="QRadioButton" name="rbCD">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>83</y>
                  <width>381</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Reconnaissance et comptage de doigts</string>
                </property>
               </widget>
               <widget class="QRadioButton" name="rbCS">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>113</y>
                  <width>381</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Contrôle de souris</string>
                </property>
               </widget>
               <widget class="QRadioButton" name="rbNONE">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>26</y>
                  <width>411</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Aucune</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="btnStart">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Démarrer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnSave">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Enregistrer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnStop">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Arrêter</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Camera</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QFrame" name="frCamera">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <action name="action_save_path">
   <property name="text">
    <string>Répertoire de sauvegarde</string>
   </property>
  </action>
  <action name="action_about_us">
   <property name="text">
    <string>A propos de nous</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionESS">
   <property name="text">
    <string>ESS</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
